components/tidb_query/src/batch/executors/fast_hash_aggr_executor.rs

```diff
@@ -407,7 +409,7 @@ mod tests {
         let exec_fast = |src_exec| {
             Box::new(BatchFastHashAggregationExecutor::new_for_test(
                 src_exec,
-                group_by_exp.clone(),
+                group_by_exp(),
```

Comment:
why does this PR still contains changes from https://github.com/tikv/tikv/pull/5763?